data/text/197.txt

```diff
@@ -488,7 +488,7 @@ The opposing {STRVAR_1 1, 0, 0} became\nthe center of attention!
 {STRVAR_1 1, 0, 0} began\ncharging power!
 The wild {STRVAR_1 1, 0, 0} began\ncharging power!
 The opposing {STRVAR_1 1, 0, 0} began\ncharging power!
-{STRVAR_1 6, 0, 0} turned into\n{STRVAR_1 6, 1, 0}!
+Nature Power turned into\n{STRVAR_1 6, 1, 0}!
```

Comment:
what was the issue here?

Reply:
That's just how it is in later gens, they hard coded it similar to Reflect, etc.